data/src/main/java/io/gnosis/data/utils/DateUtils.kt

```diff
@@ -0,0 +1,3 @@
+package io.gnosis.data.utils
+
+fun String.formatBackendDate() = this.replace("Z", "").replace("T", ", ").split(".").first()
```

Comment:
I agree that this is hacky. But it wasn't introduced in this PR, I'd prefer to fix this in a separate PR. 
I expect some bigger discussions on how to solve this.

Reply:
Yes, this is beyond the scope of this issue. The date formatting of the current backend is something that hasn't change since the prototyping days. I will try to push for this in the rust gateway service repo.

Reply:
please open an issue for this

Reply:
https://github.com/gnosis/safe-android/issues/673